src/hooks/useAiChatbot.ts

```diff
@@ -1,286 +1,6 @@
-import { useSquid } from './useSquid';
-
-export type ChatMessage = {
-  id: string;
-  type: 'ai' | 'user';
-  message: string;
-  voiceFile?: File;
-};
-
-interface AiHookResponse {
-  /**
-   * Method to initiate a chat with the AI.
-   * @param {string} prompt - The input prompt to send to the AI.
-   * @param {AiChatbotChatOptions} [options] - Optional configurations for the chat.
-   */
-  chat: (prompt: string, options?: AiChatbotChatOptions) => void;
-
-  /**
-   * Method to transcribe an audio file and then chat with the AI.
-   * @param {File} fileToTranscribe - The audio file to transcribe.
-   * @param {AiChatbotChatOptions} [options] - Optional configurations for the chat.
-   */
-  transcribeAndChat: (fileToTranscribe: File, options?: AiChatbotChatOptions) => void;
-
-  /**
-   * Method to initiate a chat with the AI and receive a voice response.
-   * @param {string} prompt - The input prompt to send to the AI.
-   * @param {Omit<AiChatbotChatOptions, 'smoothTyping'>} [options] - Optional configurations for the chat.
-   */
-  chatWithVoiceResponse: (prompt: string, options?: Omit<AiChatbotChatOptions, 'smoothTyping'>) => void;
-
-  /**
-   * Method to transcribe an audio file, chat with the AI, and receive a voice response.
-   * @param {File} fileToTranscribe - The audio file to transcribe.
-   * @param {Omit<AiChatbotChatOptions, 'smoothTyping'>} [options] - Optional configurations for the chat.
-   */
-  transcribeAndChatWithVoiceResponse: (
-    fileToTranscribe: File,
-    options?: Omit<AiChatbotChatOptions, 'smoothTyping'>,
-  ) => void;
-
-  /**
-   * History of chat messages.
-   */
-  history: ChatMessage[];
-
-  /**
-   * Data received from the AI response.
-   */
-  data: string;
-
-  /**
-   * Loading state indicating whether an operation is in progress.
-   */
-  loading: boolean;
-
-  /**
-   * Error encountered during an operation, if any.
-   */
-  error: any;
-
-  /**
-   * Indicates whether the current operation is complete.
-   */
-  complete: boolean;
-}
-
-/**
- * Custom hook for making AI queries with a given database integration ID.
- * @param integrationId - The unique identifier for the database integration instance.
- * @returns An object containing methods and state for AI chat interactions.
- */
-export function useAiQuery(integrationId: IntegrationId): AiHookResponse {
-  return useAiHook([integrationId], true);
-}
-
-/**
- * Custom hook for making AI queries with multiple database integration IDs.
- * @param integrationIds - The unique identifiers for the database integrations.
- * @returns An object containing methods and state for AI chat interactions.
- */
-export function useAiQueryMulti(integrationIds: Array<IntegrationId>): AiHookResponse {
-  return useAiHook(integrationIds, true);
-}
+import { AiHookResponse, useAiHook } from './useAiAgent';
+import { IntegrationId } from '@squidcloud/client';
 
 export function useAiChatbot(integrationId: IntegrationId, profileId: string): AiHookResponse {
```

Comment:
@jenperson lets keep `useAiQuery` and `useAiQueryMulti` here as well so that existing versions won't break, and mark them as `@deprecated`.